contributing.rst

```diff
@@ -528,6 +528,20 @@ You must include a PR comment. Things to include:
 
 Complete the pull request. The maintainers will review it as quickly as possible. If there are any problems the maintainers can't deal with, they will reach out to you.
 
+.. note::
+
+   If you happen to rename any document file(:file:`*.rst`), then be sure that you add the redirect in your PR.
+
+   To add the redirect go to :file:`s3_website.yml` and uncomment the **redirects:** line. Add all the mapppings from old file name to new file name below the **redirects:** line one by one.
```

Comment:
uncomment all of them one by one?

Reply:
I meant to say uncomment just the first redirects line and then add the redirects.
![selection_002](https://user-images.githubusercontent.com/15226041/31349767-4055bd6a-ad42-11e7-976b-ab131d0a4118.png)
![selection_003](https://user-images.githubusercontent.com/15226041/31349773-495689f8-ad42-11e7-8a8b-4e52dc483593.png)

Reply:
@adammichaelwood can you please check this and guide me what changes do I need to make.

Reply:
I would add a sentence, and change a couple words.

> Add a mapping from the old file name to the new file name below the **redirects:** line, one mapping per line. Several examples of how to format these are shown in the comments.